Changelog

```diff
@@ -5,6 +5,45 @@ py-amqp is fork of amqplib used by Kombu containing additional features and impr
 The previous amqplib changelog is here:
 http://code.google.com/p/py-amqplib/source/browse/CHANGES
 
+.. _version-2.2.0
+
+2.2.0
+=====
+:release-date: TBD
+:release-by: Ask Solem
+
+- Fix random delays in task execusion.
+
+  This is a bug that caused performance issues due to polling timeouts that occur when receiving incomplete AMQP frames. (Issues #3978 #3737 #3814) 
+  
+  Fix contributed by **Robert Kopaczewski**
+
+- Calling ``conn.collect()`` multiple times will no longer raise an ``AttributeError`` when no channels exist.
+
+  Fix contributed by **Gord Chung**
+
+- Fix compatibility code for Python 2.7.6.
+
+  Fix contributed by **Jonathan Schuff**
+
+- When running in Windows, py-amqp will no longer use the unsupported TCP option TCP_MAXSEG.
+
+  Fix contributed by **Tony Breeds**
+  
+- Added support for setting the SNI hostname header.
+
+  The SSL protocol version is now set to SSLv23
```

Comment:
I'm wondering the same as well. Should we revert this and recreate a pull request?